tests/test_analysis.py

```diff
@@ -0,0 +1,95 @@
+import covasim as cv
+import numpy as np
+
+
+# Suite of tests to test basic functionality of the analysis.py file
+# Runtime: 5.004 seconds
+def test_analysis_snapshot():
+    sim = cv.Sim(analyzers=cv.snapshot('2020-04-04', '2020-04-14'))
+    sim.run()
+    snapshot = sim['analyzers'][0]
+    people1 = snapshot.snapshots[0]            # Option 1
+    people2 = snapshot.snapshots['2020-04-04'] # Option 2
+    people3 = snapshot.get('2020-04-14')       # Option 3
+    people4 = snapshot.get(34)                 # Option 4
+    people5 = snapshot.get()                   # Option 5
+    # people3 = []  # uncomment to verify error
+    peoples = [people1, people2, people3, people4, people5]
+    for i, people in enumerate(peoples):
+        optionNum = i+1
+        assert len(people) > 0, f"Option {optionNum} should have more than 0 members"
+
+
+
+def test_analysis_hist():
+    # raising multiple histograms to check windows functionality
+    day_list = ["2020-03-30", "2020-03-31", "2020-04-01"]
+    age_analyzer = cv.age_histogram(days=day_list)
+    sim = cv.Sim(analyzers=age_analyzer)
+    sim.run()
+    assert age_analyzer.window_hists == None
+
+    # checks to make sure dictionary form has right keys
+    agehistDict = sim['analyzers'][0].get()
+    assert len(agehistDict.keys()) == 5 # TODO: is there a way to know what keys to expect?

```

Comment:
Probably OK to not do this assert, seems like it might be brittle to implementation details

Reply:
Could change to greater than something. Is there an acceptable minimum here? (Thinking 2)

Reply:
I think we can probably do without this test, if windowing and plotting works and this fails, then it would just be because the implementation has changed -- e.g., maybe this will be a list instead of a dict in future

Reply:
See above comment, as long as we keep the assertions on lines 39 and 43 I think this is probably fine. Different tests that don't necessarily go into a code coverage run can check the other parts of this feature.